src/runtime/typemanager.cs

```diff
@@ -311,6 +337,24 @@ internal static IntPtr WriteMethodDefSentinel(IntPtr mdef)
             return WriteMethodDef(mdef, IntPtr.Zero, IntPtr.Zero, 0, IntPtr.Zero);
         }
 
+        internal static void FreeMethodDef(IntPtr mdef)
+        {
+            unsafe
```

Comment:
Have we verified this still works in Unity with default options? Default Unity settings block unsafe code. I think it's just at compile-time, so unsafe compiled code is fine, but I want to make sure.

Reply:
Yes, it does, there are other uses of unsafe code in the project. (XDecref for instance)